### ISSUE TITLE
NERD tree seems to cause issues

### ISSUE THREAD

**Issue by lars-vc** [thaerkh/vim-workspace#51]
When using this plugin with NERDtree the extension fails to load the tabs correctly
![image](https://user-images.githubusercontent.com/57136208/148181408-6a6936b1-04d7-4f1c-bd72-843df8fc48b1.png)
this is the session file
```
let SessionLoad = 1
let s:so_save = &g:so | let s:siso_save = &g:siso | setg so=0 siso=0 | setl so=-1 siso=-1
let v:this_session=expand("<sfile>:p")
silent only
silent tabonly
cd ~/.config/nvim
if expand('%') == '' && !&modified && line('$') <= 1 && getline(1) == ''
  let s:wipebuf = bufnr('%')
endif
set shortmess=aoO
argglobal
%argdel
edit init.vim
let s:save_splitbelow = &splitbelow
let s:save_splitright = &splitright
set splitbelow splitright
wincmd _ | wincmd |
vsplit
1wincmd h
wincmd w
let &splitbelow = s:save_splitbelow
let &splitright = s:save_splitright
wincmd t
let s:save_winminheight = &winminheight
let s:save_winminwidth = &winminwidth
set winminheight=0
set winheight=1
set winminwidth=0
set winwidth=1
exe 'vert 1resize ' . ((&columns * 31 + 95) / 190)
exe 'vert 2resize ' . ((&columns * 158 + 95) / 190)
argglobal
enew
file NERD_tree_1
setlocal fdm=manual
setlocal fde=0
setlocal fmr={{{,}}}
setlocal fdi=#
setlocal fdl=0
setlocal fml=1
setlocal fdn=20
setlocal nofen
wincmd w
argglobal
balt init.vim
setlocal fdm=manual
setlocal fde=0
setlocal fmr={{{,}}}
setlocal fdi=#
setlocal fdl=0
setlocal fml=1
setlocal fdn=20
setlocal fen
silent! normal! zE
let &fdl = &fdl
let s:l = 1 - ((0 * winheight(0) + 22) / 44)
if s:l < 1 | let s:l = 1 | endif
keepjumps exe s:l
normal! zt
keepjumps 1
normal! 0
wincmd w
2wincmd w
exe 'vert 1resize ' . ((&columns * 31 + 95) / 190)
exe 'vert 2resize ' . ((&columns * 158 + 95) / 190)
tabnext 1
badd +0 init.vim
if exists('s:wipebuf') && len(win_findbuf(s:wipebuf)) == 0 && getbufvar(s:wipebuf, '&buftype') isnot# 'terminal'
  silent exe 'bwipe ' . s:wipebuf
endif
unlet! s:wipebuf
set winheight=1 winwidth=20 shortmess=filnxtToOF
let &winminheight = s:save_winminheight
let &winminwidth = s:save_winminwidth
let s:sx = expand("<sfile>:p:r")."x.vim"
if filereadable(s:sx)
  exe "source " . fnameescape(s:sx)
endif
let &g:so = s:so_save | let &g:siso = s:siso_save
nohlsearch
doautoall SessionLoadPost
unlet SessionLoad
" vim: set ft=vim :
```

**Comment by lars-vc** [comment]
This appears to be a issue related to NERDtree rather then this plugin
Possible fix:
https://stackoverflow.com/questions/4226905/vim-nerdtree-not-recovered-in-session-restore

**Comment by lars-vc** [comment]
Did some more googling and found fixes

**Comment by nietaki** [comment]
@lars-vc What's your fix?

**Comment by lars-vc** [comment]
Oof it has been a bit long, so I don't really remember. I think I might have been this commit to my dotfiles repo 
https://github.com/lars-vc/neovim-config/commit/2d068ff66dd6e39a614be04fede5579023bc9dc7
But I don't think I stuck long with sessions. In the end, I changed my workflow using tmux, so my session stay alive a lot longer. I also started using harpoon.nvim, so I don't need to reopen stuff anyways.
